lang/bn.json

```diff
@@ -0,0 +1,90 @@
+{
+    "Audio Player": "অডিও প্লেয়ার",
+    "Video Player": "ভিডিও প্লেয়ার",
+    "Play": "প্লে করুন",
+    "Pause": "বিরাম",
+    "Replay": "রিপ্লে করুন",
+    "Current Time": "বর্তমান সময়",
+    "Duration": "ব্যাপ্তিকাল",
+    "Remaining Time": "অবশিষ্ট সময়",
+    "Stream Type": "স্ট্রিমের ধরন",
+    "LIVE": "লাইভ",
+    "Seek to live, currently behind live": "লাইভ দেখুন, বর্তমানে লাইভের পিছনে আছে",
+    "Seek to live, currently playing live": "লাইভ দেখুন, বর্তমানে লাইভে আছে",
+    "Loaded": "লোড হয়েছে",
+    "Progress": "প্রোগ্রেস",
+    "Progress Bar": "প্রোগ্রেস বার",
+    "progress bar timing: currentTime={1} duration={2}": "{২} এর {১}",
```

Comment:
These tokens need to remains as exactly `{1}` and `{2}` for the replacement to work.

Reply:
Like this:

```suggestion
    "progress bar timing: currentTime={1} duration={2}": "{2} এর {1}",
```